docs/source/manual/openfpga_shell/openfpga_commands/fpga_bitstream_commands.rst

```diff
@@ -59,6 +59,10 @@ build_architecture_bitstream
 
     Show verbose log
 
+  .. option:: --fix_unmapped_mux_selection
```

Comment:
Should we rename the option to ``prefer_unused_input``?

Reply:
Changed to `prefer_unused_mux_input`